test/ttmlir/Decomposition/TTIR/dot_general/dot_general_test2.mlir

```diff
@@ -0,0 +1,13 @@
+// RUN: ttmlir-opt --ttir-to-ttir-decomposition %s | FileCheck %s
+module @jit_dot_general {
+  func.func public @test_dot_general(%arg0: tensor<4x10x1xf32>, %arg1: tensor<4x10x2xf32>) -> tensor<1x2xf32> {
+    %0 = "ttir.dot_general"(%arg0, %arg1) <{batchdims_a = array<i64>, batchdims_b = array<i64>, contractdims_a = array<i64: 0, 1>, contractdims_b = array<i64: 0, 1>}> : (tensor<4x10x1xf32>, tensor<4x10x2xf32>) -> tensor<1x2xf32>
+    // CHECK: "ttir.permute"
+    // CHECK: <{permutation = array<i64: 2, 0, 1>}> : (tensor<4x10x1xf32>, tensor<1x4x10xf32>) -> tensor<1x4x10xf32>
```

Comment:
`CHECK-SAME` instead of `CHECK`. With `CHECK-SAME` you can also split some checks (for example attributes and types) on different lines, which will make it more readable, and it will be more informative when it fails.